conf/reef/baremetal/mero_4_node_4_client_conf

```diff
@@ -0,0 +1,132 @@
+---
+globals:
+  - ceph-cluster:
+      name: ceph
+      networks:
+        public:
+          - 10.8.128.0/21
+      nodes:
+        - hostname: mero006
+          id: node1
+          ip: 10.8.129.226
+          role:
+            - _admin
+            - installer
+            - mon
+            - mgr
+            - rgw
+            - osd
+          root_password: passwd
+          volumes:
+            - /dev/sda
+            - /dev/sdb
+            - /dev/sdc
+            - /dev/sdd
+            - /dev/sde
+            - /dev/sdf
+            - /dev/sdg
+            - /dev/sdh
+            - /dev/sdi
+            - /dev/sdj
+            - /dev/sdk
+            - /dev/nvme0n1
+            - /dev/sdl
+        - hostname: mero008
+          id: node2
+          ip: 10.8.129.228
+          role:
+            - mds
+            - mgr
+            - rgw
+            - mon
+            - osd
+            - _admin
+          root_password: passwd
+          volumes:
+            - /dev/sda
+            - /dev/sdb
+            - /dev/sdc
+            - /dev/sdd
+            - /dev/sde
+            - /dev/sdf
+            - /dev/sdg
+            - /dev/sdh
+            - /dev/sdi
+            - /dev/sdj
+            - /dev/sdk
+            - /dev/nvme0n1
+            - /dev/sdl
+        - hostname: mero010
+          id: node3
+          ip: 10.8.129.230
+          role:
+            - mds
+            - mon
+            - rgw
```

Comment:
Let's remove rgw from node3 and node4.

This change is to make sure that we have daemon colocation recommendations met.
https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/6/html/installation_guide/red-hat-ceph-storage-considerations-and-recommendations